test/fixtures/tsconfig.default.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Let's move those files in some type test specific folder. Maybe just `<root>/type-test`